oap-server/server-receiver-plugin/zipkin-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/zipkin/handler/SpanProcessor.java

```diff
@@ -69,7 +69,7 @@ void convert(ZipkinReceiverConfig config, SpanBytesDecoder decoder, HttpServletR
     private InputStream getInputStream(HttpServletRequest request) throws IOException {
         InputStream requestInStream;
 
-        String headEncoding = request.getHeader("accept-encoding");
+        String headEncoding = request.getHeader("Content-Encoding");
```

Comment:
@adriancole which is the right name?

Reply:
content-encoding is for the incoming payload. there's no format defined for the response sent back to the client as we just send status 202. iotw, accept-encoding isn't that relevant.

Reply:
@adriancole I forgot where this code from about accept-encoding. But I remember I copied from somewhere.

So this change is right fix, right?

Reply:
maybe rename headEncoding to requestEncoding to make clear, but yeah.

It is common bug to flip these

Reply:
@credit2go-admin Could you change the variable name too?